docs/widgets/tabs.md

```diff
@@ -0,0 +1,67 @@
+# Tabs
+
+Displays a number of tab headers which may be activated with click or navigated with cursor keys.
+
+- [x] Focusable
+- [ ] Container
+
+Construct a `Tabs` widget with strings or [Text][rich.text.Text] objects as positional arguments, which will set the label in the tabs. Here's an example with three tabs:
+
+```python
+def compose(self) -> ComposeResult:
+    yield Tabs("First tab", "Second tab", Text.from_markup("[u]Third[/u] tab"))
+```
+
+This will create [Tab][textual.widgets.Tab] widgets internally, with an auto-incrementing `id` attribute (`"tab-1"`, `"tab-2"` etc).
```

Comment:
```suggestion
This will create [Tab][textual.widgets.Tab] widget internally, with an auto-incrementing `id` attribute (`"tab-1"`, `"tab-2"` etc).
```